recipes/dependency.rb

```diff
@@ -17,13 +17,13 @@
     end
 when 'ubuntu'
     case node[:platform_version]
-    when '10.04', '12.04', '14.04'
+    when '10.04', '12.04', '14.04' , '16.04'
         package 'lsb-release' do
             action :install
         end
         include_recipe 'teagent::repository'
     else
-        Chef::Application.fatal!('Only Ubuntu 10.4 (lucid), 12.04 (precise) and 14.04(trusty) are supported. Please contact support.')
+        Chef::Application.fatal!('Only Ubuntu 10.4 (lucid), 12.04 (precise), 14.04(trusty) and 16.04 (xenial) are supported. Please contact support.')
```

Comment:
Actually, we are not supporting 10.04 and 12.04 anymore, can you update this PR and also the puppet-teagent repo to show that message for these unsupported versions?